packages/website/docs/_components_pages/main/Calendar/Calendar.mdx

```diff
@@ -45,4 +46,8 @@ Discover all the available <Link to={useBaseUrl("/components/CalendarLegend#cale
 
 ### Timezones
 You can set to the configuration the preferred time zone, such as: Asia/Tokyo, Pacific/Apia, Asia/Kolkata, Europe/Sofia and etc.
-<CalendarInDifferentTimezone />
\ No newline at end of file
+<CalendarInDifferentTimezone />
+
+### CalendarWeekNumbering
+You can use the component the preferred week numbering and first day of week.
```

Comment:
Very minor one: 
`You can use the component, the preferred week numbering and first day of week.`